spec/bcm2835_spec.rb

```diff
@@ -0,0 +1,132 @@
+require 'spec_helper'
+
+
+describe 'Bcm2835' do
+  
+  before :context do
+    Platform.driver
+  end
+  
+  after :context do
+    Bcm2835.exported_pins.delete_if { true }
+  end
+
+  # describe  'GPIO' do
+    let(:file_like_object) { double("file like object") }
+
+    before :example do
+      allow(File).to receive(:read).and_return("1")
+      allow(File).to receive(:write).and_return("1")
+      allow(File).to receive(:open).and_return(file_like_object)
+    end
+
+    it '#export' do
```

Comment:
Your not wrong.... but am I right ? 
Your convention as you describe it, is really the better way to do it. My notation is also a common way, as there are some lazy guys like me around here.
can't change it now. you merge or I do not by the end of the week.